packages/cli/src/commands/validator/register.ts

```diff
@@ -26,6 +30,21 @@ export default class ValidatorRegister extends BaseCommand {
     const validators = await this.kit.contracts.getValidators()
     const accounts = await this.kit.contracts.getAccounts()
 
+    if (!res.flags.yes) {
+      const requirements = await validators.getValidatorLockedGoldRequirements()
+      const duration = requirements.duration.toNumber() * 1000
+      const check = await cli.prompt(
```

Comment:
You might want to use `binaryPrompt` from the `cli` util library instead https://github.com/celo-org/celo-monorepo/blob/master/packages/cli/src/utils/cli.ts#L79